cmake/Modules_CUDA_fix/upstream/FindCUDA.cmake

```diff
@@ -993,6 +993,9 @@ if(NOT CUDA_VERSION VERSION_LESS "3.2")
     find_cuda_helper_libs(nvcuvid)
   endif()
 endif()
+if(CUDA_cusparse_LIBRARY)
+  list(APPEND CUDA_LIBRARIES ${CUDA_cusparse_LIBRARY})
+endif()
```

Comment:
this change against policy: should not modify FindCUDA directly instead should open PR to upstream.
trying to use a workaround instead of altering CUDA_LIBRARIES directly